docs/core/tools/dotnet-workload.md

```diff
@@ -0,0 +1,43 @@
+---
+title: dotnet workload command
+description: Learn about the base dotnet workload command. Workloads allow you to manage and install optional components of .NET.
+ms.date: 10/24/2022
+---
+# dotnet workload command
+
+**This article applies to:** ✔️ .NET Core 7.0 SDK and later versions
+
+## Name
+
+`dotnet workload` - The generic driver for the .NET workloads experience.
+
+## Synopsis
+
+To get information about the available workload commands and or installed workloads:
+
+```dotnetcli
+dotnet workload [--info]
+
+dotnet workload -?|-h|--help
+```
+
+## Description
+
+The `dotnet workload` command provides commands for working with .NET workloads.
+
+  For example, [`dotnet workload install`](dotnet-workload-install.md) installs a particular workload. Each command defines its own options and arguments. All commands support the `--help` option for printing out brief documentation about how to use the command.
+
+## Options
+
+- **`--info`**
+
+  Prints out detailed information about installed workloads, including their installation source, manifest version, manifest path, and install type.
+
+- **`-?|-h|--help`**
+
+  Prints out a list of available commands.
+
+## See also
+
+- [Installing a .NET Workload](dotnet-workload-install.md)
+- [Find the available workloads on your machine](dotnet-workload-search.md)
```

Comment:
```suggestion
- [Install a workload](dotnet-workload-install.md)
- [List the installed workloads on your machine](dotnet-workload-search.md)
```

Reply:
workload search does not list the installed workloads on your machine, that is dotnet list. Dotnet search shows all workloads. Maybe we should show all 3?

Reply:
>Maybe we should show all 3

Yes. "find available" doesn't sound right to me with "on your machine"; "available" implies available to install, which means it isn't on your machine yet.